Grammar review: Installation alternatives (#5363)
wazuh/wazuh-documentation: source/deployment-options/deploying-with-ansible/guide/install-wazuh-agent.rst

.. note::

	- 	SSH key-pairing should already be configured between the ansible deployment server and the endpoints.

> 	- 	The endpoints where the agent will be deployed and added to the Ansible hosts file under the ``[wazuh-agents]`` hosts group.

Reviewer: ```suggestion
	- 	Add the endpoints where the agent will be deployed in the Ansible hosts file under the ``[wazuh-agents]`` hosts group.
```
